### PR TITLE
kube-aws: fix missing cluster-name in status cmd

### LINKED ISSUE #352: kube-aws: status prints empty cluster name
```
» kube-aws status
Cluster Name:
Controller IP:  <...>
```

### PR DESCRIPTION
**Pull request by cgag**
Info() never actually set the info.Name field to anything.

fixes #352

### CODE REVIEW

File: multi-node/aws/cmd/kube-aws/command_validate.go [hunk 1]
```diff
@@ -25,7 +25,12 @@ var (
 
 func init() {
 	cmdRoot.AddCommand(cmdValidate)
-	cmdValidate.Flags().BoolVar(&validateOpts.awsDebug, "aws-debug", false, "Log debug information from aws-sdk-go library")
```
Comment (aaronlevy) [inline multi-node/aws/cmd/kube-aws/command_validate.go]: nit: I prefer oneliners for flags (more readable if there are multiple flags, so in this case not huge deal)

File: multi-node/aws/pkg/cluster/cluster.go [hunk 2]
```diff
@@ -144,36 +144,19 @@ func (c *Cluster) Update(stackBody string) (string, error) {
...
-			break
-		}
+	resp, err := c.svc.DescribeStackResource(
+		&cloudformation.DescribeStackResourceInput{
+			LogicalResourceId: aws.String("EIPController"),
+			StackName:         c.clusterName,
+		},
+	)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get public IP of controller instance")
```
Comment (aaronlevy) [inline multi-node/aws/pkg/cluster/cluster.go]: Probably helpful to include the actual error text in the returned error

**Comment by aaronlevy** [comment]
Couple minor comments.

However, I believe this might conflict with some changes proposed in https://github.com/coreos/coreos-kubernetes/pull/346 -- may need to coordinate with @colhom on timing.

**Comment by cgag** [comment]
Rebased on Colin's #346 changes and added the aws error message to the error output. 

Some minor changes will need to be made after #362 is merged.

**Comment by colhom** [comment]
LGTM for me.

**Comment by colhom** [comment]
@cgag are we blocking for some reason on #362 before merging this one?

**Comment by cgag** [comment]
I don't think so.  I think I made that comment when I had moved some stuff around, but that's been undone.

**Pull request merged**